src/statistic/calculator.js

```diff
@@ -0,0 +1,33 @@
+const DAY_MS = 8.64e+7;
```

Comment:
seems like magic, but actually why not using 24*1000*1000*60 etc. string?

Reply:
Agreed, I'll update it.